lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -543,9 +545,9 @@ describe('GatherRunner', function() {
       ],
     };
 
-    return GatherRunner.afterPass({url, driver, passConfig}, {TestGatherer: []}).then(vals => {
+    return GatherRunner.afterPass({url, driver, passConfig}, {TestGatherer: []}).then(passData => {
```

Comment:
ok to revert these lines now?

Reply:
I think `passData` is more clear than `vals`.